core/modules/layout/js/layout.admin.js

```diff
@@ -90,6 +90,34 @@ Backdrop.behaviors.layoutDisplayEditor = {
         forcePlaceholderSize: true
       });
 
+    $('.layout-editor-block').bind('keydown', function(event) {
+        if(event.which == 38) {
```

Comment:
@docwilmot these "magic" numbers are corresponding to keyboard keys (tab/up/down). Right? Can we either assign them to variables with meaningful names or add inline comments to explain?